src/index.spec.js

```diff
@@ -1,21 +1,22 @@
-import { coerce, coerceArray } from './index';
+import { coerce, coerceArray, coerceArrayParameter, coerceParameter } from './index';
 
 const ERROR_MESSAGE = 'You failing the hardest';
-class FakeClassForTesting {}
+class FakeClassForTesting { }
 class FailingClassForTesting {
     constructor() {
         throw new Error(ERROR_MESSAGE);
     }
 }
+
 describe('coerce', () => {
     it('should be a function', () => {
         expect(coerce).toEqual(jasmine.any(Function));
     });
 
     it('should return value if it is an instanceof Type', () => {
-        const value = 'strings are fun';
+        const value = new FakeClassForTesting('strings are fun');
 
-        expect(coerce(value, String, '')).toEqual(value);
+        expect(coerceParameter(value, FakeClassForTesting, '')).toBe(value);
```

Comment:
I like the change to test for a different Type, but shouldn't this be testing `coerce` in this describe block?

Reply:
Yes, this was a miss. Addressed in a40410b